server/viame_server/viame_detection.py

```diff
@@ -274,16 +277,32 @@ def get_clip_meta(self, folder):
             required=True,
             level=AccessType.READ,
         )
-        .jsonParam("tracks", "", paramType="body")
+        .jsonParam(
+            "tracks",
+            "upsert and delete tracks",
+            paramType="body",
+            requireObject=True)
```

Comment:
I could use a jsonSchema here to describe the general shape, but `dacide.from_dict` will just do the valiadation again, and all the keys are optional, and nothing in the swagger client will display it, so I don't see the point.